File: EJ7/Rectangulo.php
<?php
    class Rectangulo {
        public $base;
        public $altura; 

        public function calcularArea(){
            $total = $this->base * $this->altura;
            echo "El área del cuadrado con base " .$this->base. " y altura " .$this->altura. " es: " .$total. "\n";
        }
    }

    $persona = new Rectangulo();
    $persona->base = "5";
    $persona->altura = "5";

    $persona->calcularArea();
?>

File: Ejercicio4/EJ10/misPaquetes/Rectangulo.java
package misPaquetes;

public class Rectangulo extends Figura {
	double base;
	double altura;
	
	public Rectangulo(double base, double altura) {
		this.base = base;
		this.altura = altura;
	}
	
	@Override
	public void calcularArea() {
		double resultado = base * altura;
		System.out.println("El resultado es: " + resultado);
	}
}
